### ISSUE TITLE
Can you share the angle threshold of the weights in the gaze correction system?

### ISSUE THREAD

**Issue by Felaim** [chihfanhsu/gaze_correction#3]
Hi, your work is excellent. 
Recently, I want to re-train your model. When I use your weights, the result is reasonable. I wonder what the angle threshold in your weights? For example, **sur_agl_limit/dif_agl_limit_v/dif_agl_limit_h**?

**Comment by chihfanhsu** [comment]
These values are used to pre-train the network for reasonable angles in live video communication and are defined in the load_dataset2.py to control the batch generator.

In our experiment, we emperically set these values as
sur_agl_limit = 16
dif_agl_limit_v = 21
dif_agl_limit_h = 26

**Comment by Felaim** [comment]
I used your parameter settings, but there was still a problem with the eyelids being unsmoothed during the test. Notice that you have chosen two participants as the validation set. I don't know which two you selected?

**Comment by chihfanhsu** [comment]
In my experiment, users for validation are 0008 and 0025. I also tried other validation pairs before, the result is highly similar.
Besides, sometimes the model won't converge well because of the small batch size and random feeding sequence.
Maybe you can try to retrain the model for few times to obtain good results. In my practice, I merely retrain the model for two times.

**Comment by Felaim** [comment]
Thank you very much, I will try more times.
